Packs/CloudShare/Integrations/CloudShare/CloudShare.py

```diff
@@ -0,0 +1,1254 @@
+
+
+def get_env_multiple_resources_command(client, args):
+    res = client.send_request(
+        'GET',
+        'envs/actions/getmultipleenvsresources',
+        queryParams={k: v for k, v in args.items()}
+    )
+    if res.status == 200:
+        md = tableToMarkdown('CloudShare Environment Resources from {args.starttime} to {args.endtime}:', res.content)
+        command_results = CommandResults(
+            outputs_prefix="CloudShare.EnvironmentResources",
+            outputs_key_field='id',
+            outputs=res.content if res.content else None,
+            readable_output=md
+        )
+        return_results(command_results)
+    else:
+        return_error(f"Error getting environment resources - {res.content}")
+
+
+def extend_env_command(client, args):
+    envId = args.get('envId')
+    res = client.send_request(
+        'PUT',
+        'envs/actions/extend',
+        queryParams={"envId": envId}
+    )
+    if res.status == 200:
+        return_results(f"Postpone environment {envId} suspend successful")
+    else:
+        return_error(f"Error postponing environment {envId} suspension- {res.content}")
+
+
+def postpone_env_suspend_command(client, args):
+    envId = args.get('envId')
+    res = client.send_request(
+        'PUT',
+        'envs/actions/postponeinactivity',
+        queryParams={"envId": envId}
+    )
+    if res.status == 200:
+        return_results(f"Extend environment {envId} successful")
+    else:
+        return_error(f"Error extended environment {envId} - {res.content}")
+
+
+def resume_env_command(client, args):
+    envId = args.get('envId')
+    res = client.send_request(
+        'PUT',
+        'envs/actions/resume',
+        queryParams={"envId": envId}
+    )
+    if res.status == 200:
+        return_results(f"Environment {envId} resumed successfully")
+    else:
+        return_error(f"Error resuming environment {envId} - {res.content}")
+
+
+def revert_env_command(client, args):
+    envId = args.get('envId')
+    snapshotId = args.get('snapshotId')
+    res = client.send_request(
+        'PUT',
+        'envs/actions/revert',
+        queryParams={"envId": envId, "snapshotId": snapshotId}
+    )
+    if res.status == 200:
+        return_results(f"Environment {envId} reverted to snapshot {snapshotId} successfully")
+    else:
+        return_error(f"Error reverting environment {envId} to snapshot {snapshotId} - {res.content}")
+
+
+def suspend_env_command(client, args):
+    envId = args.get('envId')
+    res = client.send_request(
+        'PUT',
+        'envs/actions/suspend',
+        queryParams={"envId": envId}
+    )
+    if res.status == 200:
+        return_results(f"Environment {envId} suspended successfully")
+    else:
+        return_error(f"Error suspending environment {envId} - {res.content}")
+
+
+def get_env_command(client, args):
+    envID = args.get('envID')
+    res = client.send_request(
+        'GET',
+        f'envs/{envId}',
```

Comment:
I believe this is a mistake.
```suggestion
        f'envs/{envID}',
```